### ISSUE TITLE
Proposed SIG-Release meeting agenda for July-19-22

### ISSUE THREAD

**Issue by Ulrick28** [o3de/sig-release#71]
## Meeting Details

- **Date/Time:** July 19, 2022 @ 5:00pm UTC / 10:00am PDT
- **Location:** [Discord SIG-Release Voice Room](https://discord.gg/Z2bzwCRJEz)
- **Moderator:** @Ulrick28  
- **Note Taker** @tonybalandiuk 

## SIG Updates
**What happened since the last meeting?**
* Question was raised by the tsc concerning communicating any sort of date in the roadmap. Open source projects do not communicate feature time frames due to the volunteer nature of development. 

## Meeting Agenda
* Continue review next release details https://github.com/o3de/sig-release/issues/64 - re-review the dates, any comments?
* Continue discussion for Roadmap https://github.com/o3de/sig-release/issues/62
     * This should always be a topic during release sig meetings as the roadmap will continually change.
     * Need to edit this doc https://docs.google.com/spreadsheets/d/1U_XAw9YJoPE3FlYeBKUz4kYjFRDS09NS3_YuZSZJ2qA/edit#gid=1112481565
* Election of chair and co-chair - issue opened at https://github.com/o3de/sig-release/issues/66
     * Need a time frame that is soon as @Ulrick28 will need to step down as chair and cannot be co-chair
* Discuss adding Alpha and Beta phases to our release process, starting in 2023, Proposed by @jeremyong-az 
     * Goal is to provide known quality points to users. Many users will not use nightly builds, but would test a project integration/upgrade with a beta build. This provides additional avenues of user testing.
     * Need to determine if the impact (gain) of the alpha/beta releases worth the added complexity.
     * Need to define what determines Alpha, what determines Beta. 
     * Need to determine if this is branches or just tags in stabilization (tags will be easier to manage and less complexity, a branch is more obvious to users)
     * We would need to expose alpha and beta builds on the download page during a release phase. This will require coordination with sig-build. 
     * We would need more documentation around the alpha, beta, and final, primarily to cover feature and/or bug fix deltas.
     * We would need to develop a mechanism for users to report issues against the builds
     * Soonest we should start Alpha and Beta is 2023. This allows us time to develop a plan, get feedback, document the process, and communicate the final process. We do not want developers having to adapt 'on the fly' during the October 2022 release.

## Outcomes from Discussion topics

**Discuss outcomes from agenda**

## Action Items

**Create actionable items from proposed topics**

## Open Discussion Items

List any additional items below!

**Comment by hultonha** [comment]
Topic: Discuss next steps for deprecation RFC (https://github.com/o3de/sig-release/issues/50)
Presenter: @hultonha
Length of time: 10 minutes
Information: I'd really love to try and move the [Deprecation RFC](https://github.com/o3de/sig-release/issues/50) to an approval state. A lot of great comments were added to the doc and I've attempted to address as many as I can without the scope growing too much.

We have more and more engineers looking to deprecate and remove functionality so having a reference we can point people at for what to do is going to be incredibly helpful (and not having something concrete right now is starting to hurt us). I also want to caution against perfect being the enemy of good, nothing is set in stone and we can always refine and update the doc as we get more experience with deprecation in O3DE. Thanks!
